src/NSwag.CodeGeneration/Models/OperationModelBase.cs

```diff
@@ -350,9 +350,14 @@ protected IList<OpenApiParameter> GetActualParameters()
                 _operation?.RequestBody?.Content?.ContainsKey("multipart/form-data") == true ?
                 _operation.RequestBody.Content["multipart/form-data"]?.Schema : null;
 
-            if (formDataSchema != null && formDataSchema.ActualProperties.Count > 0)
+            var formDataProperties = formDataSchema?.ActualProperties.Count > 0
+                ? formDataSchema.ActualProperties
+                : formDataSchema?.Reference?.ActualProperties.Count > 0
```

Comment:
Do you need reference check here? I'd expect that ActualProperties resolves this automatically, no?

Reply:
It does not. When running the `When_body_is_binary_with_additional_data_via_component_ref_then_all_parameters_supplied_in_CSharp_ASPNETCore` test I added `formDataSchema?.ActualProperties.Count` is 0 but `formDataSchema?.Reference?.ActualProperties.Count` is 2.

Reply:
ActualSchema.ActualProperties works though. I will push an update to do that.